lib/workers/repository/process/lookup/index.ts

```diff
@@ -369,7 +371,7 @@ export async function lookupUpdates(
       if (versioning.valueToVersion) {
         // TODO #7154
         res.currentVersion = versioning.valueToVersion(res.currentVersion!);
-        for (const update of res.updates || []) {
+        for (const update of res.updates || /* istanbul ignore next*/ []) {
```

Comment:
is it impossible to create a test? should we use `!` assertion instead?

Reply:
It's unrelated coverage miss, I wanted to refactor it later